dappnode_package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "near.dnp.dappnode.eth",
   "version": "0.1.5",
-  "upstreamVersion": "1.36.5",
+  "upstreamVersion": "1.37.0-rc.3",
```

```suggestion
  "upstreamVersion": "1.36.5",
```
Testnet RC not applicable to mainnet